docs/pages/reference/terraform-provider/data-sources/auth_preference.mdx

```diff
@@ -40,8 +40,8 @@ Optional:
 - `message_of_the_day` (String)
 - `okta` (Attributes) Okta is a set of options related to the Okta service in Teleport. Requires Teleport Enterprise. (see [below for nested schema](#nested-schema-for-specokta))
 - `require_session_mfa` (Number) RequireMFAType is the type of MFA requirement enforced for this cluster. 0 is "OFF", 1 is "SESSION", 2 is "SESSION_AND_HARDWARE_KEY", 3 is "HARDWARE_KEY_TOUCH", 4 is "HARDWARE_KEY_PIN", 5 is "HARDWARE_KEY_TOUCH_AND_PIN".
-- `second_factor` (String) SecondFactor is the type of mult-factor.
-- `second_factors` (List of Number) SecondFactors is a list of supported second factor types.
+- `second_factor` (String) SecondFactor is the type of mult-factor. Deprecated: Prefer setting SecondFactors instead.
+- `second_factors` (List of Number) SecondFactors is a list of supported second factor types. 1 is "otp", 2 is "webauthn", 3 is "sso", If unspecified, the current default value is [1], or ["otp"].
```

Comment:
Do we have to say `List of Number`?

Reply:
This is automatically generated by `terraform-plugin-docs` - https://github.com/hashicorp/terraform-plugin-docs/blob/2004e44ee970c07f78889f7525f794c8d3654dfc/internal/schemamd/write_type.go#L51C4-L51C5